presto-main/src/main/java/com/facebook/presto/metadata/BoundVariables.java

```diff
@@ -71,9 +71,7 @@ public boolean containsLongVariable(String variableName)
     private static <T> T getValue(Map<String, T> map, String variableName)
     {
         checkState(variableName != null, "variableName is null");
-        T value = map.get(variableName);
-        checkState(value != null, "value for variable '%s' is null", variableName);
```

Comment:
Is removing this the right thing to do, or should we be calling contains? Seems like this will remove a layer of validation for functions that implement `specialize`